src/Windows.Core/DEVMODEW.cs

```diff
@@ -0,0 +1,783 @@
+        /// This value is also known as the display device's vertical refresh rate. Display drivers use this member.
+        /// It is used, for example, in the ChangeDisplaySettings function. Printer drivers do not use this member.
+        /// </summary>
+        /// <remarks>
+        /// When you call the <c>User32.EnumDisplaySettings</c> function, the <see cref="dmDisplayFrequency"/> member may return with the value 0 or 1.
+        /// These values represent the display hardware's default refresh rate.
+        /// This default rate is typically set by switches on a display card or computer motherboard, or by a configuration program that does not use display functions such as <c>User32.EnumDisplaySettings</c>.
+        /// </remarks>
+        /// <remarks>
+        /// This member is not used for printers.
+        /// </remarks>
+        [FieldOffset(120)]
+        public uint dmDisplayFrequency;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceIcmMethodOptions"/> constants.
+        /// </summary>
+        [FieldOffset(124)]
+        public DEVMODEW_DeviceIcmMethodOptions dmICMMethod;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceIcmIntentOptions"/> constants.
+        /// </summary>
+        [FieldOffset(128)]
+        public DEVMODEW_DeviceIcmIntentOptions dmICMIntent;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DeviceMediaTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(132)]
+        public DEVMODEW_DeviceMediaTypeOptions dmMediaType;
+
+        /// <summary>
+        /// Specifies one of the <see cref="DEVMODEW_DitherTypeOptions"/> constants.
+        /// </summary>
+        [FieldOffset(136)]
+        public DEVMODEW_DitherTypeOptions dmDitherType;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(140)]
+        public uint dmReserved1;
+
+        /// <summary>
+        /// Not used; must be zero.
+        /// </summary>
+        [FieldOffset(144)]
+        public uint dmReserved2;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(150)]
+        public uint dmPanningWidth;
+
+        /// <summary>
+        /// This member must be zero.
+        /// </summary>
+        [FieldOffset(154)]
+        public uint dmPanningHeight;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="DEVMODEW"/> struct
+        /// with the <see cref="dmSize" /> field initialized.
+        /// </summary>
+        /// <returns>
+        /// An instance of the structure.
+        /// </returns>
+        public static DEVMODEW Create() => new DEVMODEW { dmSize = (ushort)sizeof(DEVMODEW) };
+    }
+
+    public enum DEVMODEW_DisplayOptions : uint
+    {
+    }
+
+    public enum DEVMODEW_PrinterOptions : uint
```

Comment:
Can you add something like this above each enum defined here?

```cs
/// <summary>Defines the values that may be used in the <see cref="DEVMODE.dnNup" /> member.</summary>
```

Reply:
Will do.